packages/format-library/src/link/index.js

```diff
@@ -38,10 +38,38 @@ function Edit( {
 	onFocus,
 	contentRef,
 } ) {
-	const [ addingLink, setAddingLink ] = useState( false );
+	const [ editingLink, setEditingLink ] = useState( false );
+	const [ creatingLink, setCreatingLink ] = useState( false );
+
 	// We only need to store the button element that opened the popover. We can ignore the other states, as they will be handled by the onFocus prop to return to the rich text field.
 	const [ openedBy, setOpenedBy ] = useState( null );
 
+	const [ autoFocus, setAutoFocus ] = useState( true );
+
+	function setIsEditingLink( _val, { shouldAutoFocus = true } = {} ) {
+		setEditingLink( _val );
+		setAutoFocus( shouldAutoFocus );
+	}
+
+	function setIsCreatingLink( _val ) {
+		// Don't add a new link if there is already an active link.
```

Comment:
what is `_val`?

Reply:
I used a better name here in this place and in one other place where I used the same silly pattern. Thanks for calling it out 👍